Blockzilla/UIConstants.swift

```diff
@@ -180,7 +180,7 @@ struct UIConstants {
         static let externalAppLinkWithAppName = NSLocalizedString("externalAppLinkWithAppName.messageTitle", value: "%@ wants to open %@", comment: "Dialog title used for opening an external app from Focus. First placeholder string is the app name of either Focus or Klar and the second placeholder string specifies the app it wants to open.")
         static let open = NSLocalizedString("ExternalAppLink.openTitle", value: "Open", comment: "Button label for opening another app from Focus")
         static let cancel = NSLocalizedString("ExternalAppLink.cancelTitle", value: "Cancel", comment: "Button label used for cancelling to open another app from Focus")
-        static let photosPermissionTitle = NSLocalizedString("photosPermission.title", value: "“%@” Would Like to Access Your Photos", comment: "Dialog title used for requesting a user to enable access to Photos. Placeholder is either Firefox Focus or Firefox Klar")
+        static let photosPermissionTitle = String(format: NSLocalizedString("photosPermission.title", value: "“%@” Would Like to Access Your Photos", comment: "Dialog title used for requesting a user to enable access to Photos. Placeholder is either Firefox Focus or Firefox Klar"), AppInfo.productName)
```

Comment:
Not sure if localization can localize this, but then again Idk. But usually I would just leave the `NSLocalizedString` and then put the `String(format: ... )` when I actually use `photosPermissionTitle`, aka `String(format: photosPermissionTitle, AppInfo.productName)` just to keep it consistent